app/styles/Menubar.module.css

```diff
@@ -18,7 +18,7 @@
   background-color: #ff5050;
   color: transparent;
   width: 0%;
-  content: '.';
+  content: ".";
```

Comment:
Menubar is *not* your component.   Keep your component independent.

Any coupling should be done via the bootstrap "design language"